packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import PopperJS from 'popper.js'
```

Comment:
`PopperJS` is improperly imported here - given that namespace is imported, it should be 

```
import * as PopperJS from 'popper.js'
```

Currently the code transpiled to CommonJS will result in the following client's error:

![image](https://user-images.githubusercontent.com/9024564/58198256-37a24780-7cce-11e9-863c-d07af0c448a5.png)

Reply:
I tried updating to:
```typescript
import * as PopperJS from 'popper.js'
```

and ended up with this error:
![Screenshot 2019-05-23 at 10 12 33](https://user-images.githubusercontent.com/5442794/58236663-a8377b80-7d43-11e9-8f7e-e879aa6df7ac.png)

If I change the constructor to `new PopperJS.default` I get a lint error that constructor cannot be lowercase and the code transpiled to CommonJS will result in the same error.

Are the types wrong? I'm blocked with this right now...
@layershifter @kuzhelov

Reply:
have checked `popper.js` module used in Stardust, as well as tried to repro the issue in Stardust environment - seems that everything is working fine, as well as types are properly defined for PopperJS module. Maybe this is a problem with client environment setup - lets unblock this PR and see.

Reply:
@layershifter, what are your thoughts on that?